### DISCUSSION TITLE
How to access Lambda Context in ASP.NET Core integratoin?

### DISCUSSION THREAD

**Discussion by Lanayx** [aws/aws-lambda-dotnet#1770] [Q&A] [answered]
Hi! I've set up my lambda with ASP.NET Core integration using `.AddAWSLambdaHosting(LambdaEventSource.RestApi)` and everything works fine. However, I'd like to extract information about user that calls lambda. Without ASP.NET core integration I'm able to access user with `lambdaContext.RequestContext.Identity.User`, is it possible to get this while handling ASP.NET request? If not, will it be possible to create PR that copies this info to `HttpContext`?

**Comment by Dreamescaper** [comment, marked as answer]
It _is_ in HttpContext:
```csharp
if (context.Items.TryGetValue(AbstractAspNetCoreFunction.LAMBDA_CONTEXT, out var item)
{
...
}
```
